src/components/modals/ButtonsModal.tsx

```diff
@@ -44,6 +58,7 @@ export interface ButtonModalProps<Buttons> {
 }
 
 /**
+ * @deprecated
```

Comment:
This is the wrong Javadoc syntax. The first line is always the item description, and then the @ tags go below that. So, please move the `@deprecated` to the bottom of the block and then add a message, so `@deprecated Use ButtonsModal2 instead`